src/main/java/com/ning/http/client/cookie/Cookie.java

```diff
@@ -12,8 +12,13 @@
  */
 package com.ning.http.client.cookie;
 
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+
 public class Cookie {
 
+    protected long whenCreated;
+    private static long UNDEFINED_MAX_AGE = -9223372036854775808L;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this field "UNDEFINED_MAX_AGE" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. ([squid:S3008](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS3008))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-grizzly-ahc&pullRequest=19&issues=AWkLxe_KotIXnmsUpXss&open=AWkLxe_KotIXnmsUpXss)

[ISSUE_KEY]: AWkLxe_KotIXnmsUpXss